01-CodeReviewEx/Shop-Ex/Account.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shop_Ex
+{
+    public class Account
+    {
+        public double principal, rate; int daysActive, accountType;
```

Comment:
Deklarując nową zmienną, lepiej zapisać ją w nowym wierszu, jest to czytelniejsze.